pkg/minikube/cluster/cluster.go

```diff
@@ -523,3 +525,36 @@ func createVirtualboxHost(config MachineConfig) drivers.Driver {
 	d.HostOnlyCIDR = config.HostOnlyCIDR
 	return d
 }
+
+type vmwareDriver struct {
+	*drivers.BaseDriver
+
+	Memory         int
+	DiskSize       int
+	CPU            int
+	ISO            string
+	Boot2DockerURL string
+
+	SSHPassword    string
+	ConfigDriveISO string
+	ConfigDriveURL string
+	NoShare        bool
+}
+
+func createVmwareHost(config MachineConfig) *vmwareDriver {
+	return &vmwareDriver{
+		BaseDriver: &drivers.BaseDriver{
+			MachineName: constants.MachineName,
+			StorePath:   constants.Minipath,
+			SSHUser:     "docker",
+			SSHPort:     22,
+		},
+		Boot2DockerURL: config.GetISOFileURI(),
+		DiskSize:       config.DiskSize,
+		Memory:         config.Memory,
+		CPU:            config.CPUs,
+		ISO:            filepath.Join(constants.Minipath, "machines", constants.MachineName, "boot2docker.iso"),
+		NoShare:        true,
+		SSHPassword:    "tcuser",
```

Comment:
@anjannath why we need password here for vmware case when you can have ssh key for docker user?

Reply:
All the calls in these lines needs the ssh password, https://github.com/machine-drivers/docker-machine-driver-vmware/blob/abd4e62c279b39fdb3fec9b23e8e2b1f2abd3f60/pkg/drivers/vmware/driver.go#L299-L311